shoop/core/cache/impl.py

```diff
@@ -72,7 +71,8 @@ def bump_version(self, cache_key):
         :type cache_key: str
         """
         namespace = _get_cache_key_namespace(cache_key)
-        version = str(time.time())
+        current_version = self.get_version(cache_key) or 1
+        version = current_version + 1
```

Comment:
Should we just use Django cache's `incr_version()`? (See https://docs.djangoproject.com/en/1.8/topics/cache/#cache-versioning)